python/psautohint/fdTools.py

```diff
@@ -127,14 +131,14 @@ def getFontInfo(self):
 
     def buildBlueLists(self):
         if (self.BaselineOvershoot is None):
-            print("Error: FDDict definition %s is missing the BaselineYCoord/"
-                  "BaselineOvershoot values. These are required." %
-                  self.DictName)
+            log.warning("FDDict definition %s is missing the BaselineYCoord/"
```

Comment:
All the print("Error") in this file that got changed to log.warning() should be changed to raise FontInfoParseError

Reply:
I think this part of the code is parsing FDDict definitions that would be provided via a fontinfo file. It's better to raise an error, so much so that the actual message says _These are required_.